README.md

```diff
@@ -15,3 +15,16 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+
+The worst case for my implementation would be $\Theta$(V+E). This would be the case where the outer for loop has to iterate through every node in the graph, and the inner for loop has to iterate through every node's neighbors which would cover all the edges. With this case we would have to iterate through every node and every edge. 
```

Comment:
Aren't you doing a complete exploration of the graph each time, i.e. all nodes for every iteration?